demos/common/python/openvino/model_zoo/model_api/models/background_matting.py

```diff
@@ -0,0 +1,148 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import numpy as np
+
+from .image_model import ImageModel
+from .model import WrapperError
+
+
+class VideoBackgroundMatting(ImageModel):
```

Comment:
The classes for Background Matting should have `parameters(cls)` method. The example can be found in Deblurring wrapper, since it also does not accept any configuration parameters

Reply:
`parameters(cls)` methods are added for both classes.